cmd/agent/app/reporter/flags.go

```diff
@@ -39,12 +47,14 @@ type Type string
 
 // Options holds generic reporter configuration.
 type Options struct {
-	ReporterType Type
-	AgentTags    map[string]string
+	ReporterType        Type
+	DuplicateTagsPolicy string
+	AgentTags           map[string]string
 }
 
 // AddFlags adds flags for Options.
 func AddFlags(flags *flag.FlagSet) {
+	flags.String(duplicateTags, Client, "Jaeger Tags override policy. Accepted values are client, agent, duplicate.")
```

Comment:
This option is a bit cryptic and deserves a good CLI help text. Something like:

> The tag override policy, defining what to do when the same process tag is added by the client and the agent. Accepted values are 'client' (keep client's value), 'agent' (keep agent's value) and 'duplicate' (keep both).

This ends up here as well: https://www.jaegertracing.io/docs/1.14/cli/

Reply:
Let's also add info about what the default value is. @Pothulapati - I think @jpkrohling recommended (and I agree) that we should keep `duplicate` as default, lest we confuse users.

Reply:
The default value should be added automatically by viper. This is what I currently see when I run `--help` with the agent built from this PR:

```
--jaeger.duplicate-tags string   Jaeger Tags override policy. Accepted values are client, agent, duplicate. (default "client")
```

Reply:
Updated it to `The tag override policy, defining what to do when the same process tag is added by the client and the agent. Accepted values are 'client' (keep client's value), 'agent' (keep agent's value) and 'duplicate' (keep both).`